CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # io.js ChangeLog
 
+## 2015-05-24, Version 2.0.3, @rvagg
+
+### Notable changes
+
+* **crypto**: Diffie-Hellman key exchange (DHE) parameters (`'dhparams'`) must now be 1024 bits or longer or an error will be thrown. A warning will also be printed to the console if you supply less than 2048 bits. See https://weakdh.org/ for further context on this security concern. (Shigeki Ohtsu) [#1739](https://github.com/nodejs/io.js/pull/1739).
+* **node**: A new `--trace-sync-io` command line flag will print a warning and a stack trace whenever a synchronous API is used. This can be used to track down synchronous calls that may be slowing down an application. (Trevor Norris) [#1707](https://github.com/nodejs/io.js/pull/1707).
+* **node**: To allow for chaining of methods, the `setTimeout()`, `setKeepAlive()`, `setNoDelay()`, `ref()` and `unref()` methods used in `'net'`, `'dgram'`, `'http'`, `'https'` and `'tls'` now return the current instance instead of `undefined` (Roman Reiss) [#1779](https://github.com/nodejs/io.js/pull/1779).
```

Comment:
This change was split up in three PRs by Evan and me: https://github.com/nodejs/io.js/pull/1699 https://github.com/nodejs/io.js/pull/1768 https://github.com/nodejs/io.js/pull/1779